docs/rules/meta-property-ordering.md

```diff
@@ -12,7 +12,7 @@ This rule enforces that meta properties of a rule are placed in a consistent ord
 
 This rule has an array option:
 
-* `['type', 'docs', 'fixable', 'hasSuggestions', 'schema', 'messages', 'deprecated', 'replacedBy']` (default): The order that the properties of `meta` should be placed in.
+* `['type', 'docs', 'fixable', 'hasSuggestions', 'deprecated', 'replacedBy', 'schema', 'messages']` (default): The order that the properties of `meta` should be placed in.
```

Comment:
imo, the two props should be highlighted, what do you think change them to be in the 1st place?

Reply:
Sure, I would be open to that.